source/installation-guide/securing_api.rst

```diff
@@ -5,11 +5,18 @@
 Securing the Wazuh API
 ======================
 
-By default, the communication between the Wazuh Kibana plugin and the Wazuh API is encrypted with HTTPS. This means that if you do not provide your own private key and certificate, the Wazuh API will generate its own in the first run. In addition, the default users are ``wazuh`` and ``wazuh-wui``, and the default password is ``wazuh`` for both of them. For this reason, it is highly recommended to at least change the default passwords and to use your own certificate since the one created by the Wazuh API is self signed. These are the **recommended changes** to apply:
+The communication between the Wazuh UI and the Wazuh API is encrypted with HTTPS, which means if the users do not provide their own private key and certificate then Wazuh API will generate its own during the first run. Additionally, ``wazuh`` and ``wazuh-wui`` users are created by default having ``wazuh`` as their password. Because of that, it is very important to secure Wazuh API once it has been installed.
+
+.. warning::
+  It is highly recommended to change the default passwords and to use your own certificate since the one created by the Wazuh API is self-signed.
+
+
+Recommended changes to securize Wazuh API
+-----------------------------------------
 
 #. Modify HTTPS parameters:
 
-    The Wazuh API has HTTPS enabled by default. If you do not have any certificate in ``WAZUH_PATH/api/configuration/ssl``, it will generate the private key and the self signed certificate. If that was the case, the following lines will appear in ``WAZUH_PATH/logs/api.log``:
+    The Wazuh API has HTTPS enabled by default. In case there is no available certificate in ``WAZUH_PATH/api/configuration/ssl`` Wazuh API will generate the private key and a self-signed certificate. If that was the case, the following lines will appear in ``WAZUH_PATH/logs/api.log``:
```

Comment:
```suggestion
    The Wazuh API has HTTPS enabled by default. In case there is no available certificate in ``WAZUH_PATH/api/configuration/ssl``, the Wazuh API will generate the private key and a self-signed certificate. If that is the case, the following lines will appear in ``WAZUH_PATH/logs/api.log``:
```